constants.py

```diff
@@ -3,8 +3,15 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+TRADING_CONFIG = {
+    ('currency', 'TW'): {
+        'price_dataset': 'currency_price_tw',
+        'cost_function': thousandaire.trading_cost.cost_currency}
+}
+TRADING_REGION = ['TW']
```

Comment:
Use `TRADING_REGIONS` or `TRADING_REGION_LIST` to specify that it is a iterable.